src/client/Microsoft.Identity.Client/Platforms/iOS/EmbeddedWebview/MsalAuthenticationAgentUIViewController.cs

```diff
@@ -54,7 +54,10 @@ protected WKWebView PrepareWKWebView()
         {
             WKWebViewConfiguration wkconfg = new WKWebViewConfiguration() { };
 #if !IS_APPCENTER_BUILD
-            wkconfg.DefaultWebpagePreferences.PreferredContentMode = WKContentMode.Mobile;
+            if (UIDevice.CurrentDevice.CheckSystemVersion(13, 0))
+            {
+                wkconfg.DefaultWebpagePreferences.PreferredContentMode = WKContentMode.Mobile;
```

Comment:
I suggest that this change goes out in a different commit. No new review necessary, ship it!

Reply:
[https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1629](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1629) new PR here

---
In reply to: [378472216](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1627#discussion_r378472216) [](ancestors = 378472216)